docs/bool2.md

```diff
@@ -136,25 +136,25 @@ ___
 
 ### Minterms
 
-A minterm is defined as the product term of n variables, in which each of the n variables will appear once either in its complemented or un-complemented form. The min term is denoted as mi where i is in the range of 0 ≤ i < 2<sup>n</sup>.
+A minterm is defined as the product term of n variables, in which each of the n variables will appear once either in its complemented or un-complemented form. The minterm is denoted as mi where i is in the range of 0 ≤ i < 2<sup>n</sup>.
```

Comment:
![image](https://user-images.githubusercontent.com/14032427/84003981-77df7100-a988-11ea-979a-0f0b6dc17d0e.png)

Reply:
Good attempt, however, I think I have seen similar content somewhere. 

Please note:
- You may take the help of links given as references, other digital material, or reference books, but it doesn't mean that you `ctrl+c` and `ctrl+v` everything. Try rephrasing it, explaining the points, or adding valuable information, explanations, illustrations along with. 
- We have our own simulator, where you can realize any circuit as needed. Make use of it to make page interactive.
- We have various components in the simulator, which can be used for illustrations. Using images from some other sources may invite copyright claims.

We need to be genuine in our work. 
The CircuitVerse community is here for your help, and types of contributions are valued.

Reply:
Sorry,
In this PR, **I am actually trying to fix only the spelling mistake in the already written data.
That's the reason I didn't check for plagiarism.**

However, as you have pointed I will try to edit the content and fix that plagiarism issue too.